README.md

```diff
@@ -5,47 +5,19 @@ This repository provides tools and scripts for building and testing the vSphere
 
 The CSI driver, when used on Kubernetes, also requires the use of the out-of-tree vSphere Cloud Provider Interface [CPI](https://github.com/kubernetes/cloud-provider-vsphere).
```

Comment:
Comment on line#4. Can we say that the driver is GA but some of the features may be beta? We cannot declare a feature as GA when we have not fully qualified it or the feature itself is not GA in CSI spec or Kubernetes.

Reply:
phrased as 

> This driver is in a stable `GA` state and is suitable for production use. Some of the features may be in the `beta` phase. Please refer [feature matrix](https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html) for more details.  vSphere CSI driver requires vSphere 6.7 U3 or higher in order to operate.